src/main/java/cn/nukkit/Server.java

```diff
@@ -535,7 +536,7 @@ public Level remove(Object key) {
                 put("generator-settings", "");
                 put("level-name", "world");
                 put("level-seed", "");
-                put("level-type", "DEFAULT");
+                put("level-type", "PowerNukkitOverworld"); // Now we use PowerNukkit generator
```

Comment:
Using the new experimental generator should be opt-in.
```suggestion
                put("level-type", "DEFAULT");
```

Reply:
Okay